release/build_release_package.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-rm shopgate-magento-integration.tgz
+rm shopgate_module-*.tgz
```

Comment:
should write `-f` param as it may request "are you sure you want to delete?" without an answer on certain linux distributions where `rm` is aliased into `rm -i`. Should do it for the rest of the statements as well.